util/kube/kube.go

```diff
@@ -84,6 +84,40 @@ func GetLiveResource(dclient dynamic.Interface, obj *unstructured.Unstructured,
 	return liveObj, nil
 }
 
+// DeleteResourceWithLabel delete all resources which match to specified label selector
+func DeleteResourceWithLabel(config *rest.Config, namespace string, labelSelector string) error {
+	dynClientPool := dynamic.NewDynamicClientPool(config)
+	disco, err := discovery.NewDiscoveryClientForConfig(config)
+	if err != nil {
+		return err
+	}
+	resources, err := disco.ServerResources()
+	if err != nil {
+		return err
+	}
+	for _, apiResourcesList := range resources {
+		for _, apiResource := range apiResourcesList.APIResources {
+			deleteSupported := false
+			for _, verb := range apiResource.Verbs {
+				if verb == "deletecollection" {
+					deleteSupported = true
```

Comment:
@alexmt maybe worth adding a `break` here terminate iteration?

Reply:
Done

Reply:
Awesome!  Thank you!